### DISCUSSION TITLE
等了一周多收到nolan的激活码 但输入却显示经销商代码错误！ 还是退款吧！

### DISCUSSION THREAD

**Discussion by kennylo47** [getlantern/lantern-forum-cn#652] [General]
<img width="263" alt="截屏2024-06-27 23 25 47" src="https://github.com/getlantern/lantern-forum-cn/assets/173150461/28343885-9df16-4829-bcf5-ab1ffe2c0864">
心累！已经申请退款 请尽快安排！谢谢！

**Comment by MaxMKMax** [comment]
@kennylo47 
您好. 请问您填对邮件吗? 请您再给我们一次机会. 我帮您在解决看. 谢谢您的耐心等待. 
我会跟Nolan在沟通一下.

**Reply by kennylo47** [reply]
都试了几遍 还是没办法激活 不知道你们还可以怎么解决问题 不行就退款 谢谢

**Reply by kennylo47** [reply]
请问处理了没？ @mike777khoo

**Comment by kennylo47** [comment]
请问处理了没？ @Mike777Khoo

**Reply by MaxMKMax** [reply]
@kennylo47 请您在这里提供我支付单据和您的账号邮箱. 我帮您直接找负责人解决问题. 我帮您拿激活码! 不好意思让您久等. 

前几天我住医院了. 所以没上线. 非常抱歉啊.

**Reply by kennylo47** [reply]
![WechatIMG876](https://github.com/getlantern/lantern-forum-cn/assets/173150461/5b0c6f8a-7b12-45ce-b617-99c4135e9be9)
lawy2k47@hotmail.com
如果不行就直接退款
不然来来回回太浪费时间 谢谢

**Comment by MaxMKMax** [comment]
@kennylo47 您放心. 我一定帮您解决问题!!!

**Comment by MaxMKMax** [comment]
@kennylo47 您好. 问题解决了. 我们的客服可您重新发邮件了.

**Reply by kennylo47** [reply]
是的 第二封邮件收到 刚开始输入也是显示经销商代码错误，但重启后显示已经激活，问题目前已经解决，希望能够正常使用下去，不要再这么折腾用户了。
祝各位顺利，谢谢@mike777khoo 的协助

**Reply by kennylo47** [reply]
昨晚激活后我移除了一些以前的授权机器，结果今天早上当我再次重启蓝灯之后 又没有了专业版...再次输入激活码也没有用，可能是我误把这台机的授权删了，麻烦您这边再发一次吧谢谢@mike777khoo

**Reply by kennylo47** [reply]
中午我登录另外一台之前授权笔记本并更新软件后，发现是能登陆pro账号的！但当我希望通过链接码授权我现在主力笔记本却始终无法激活...目前显示现在只授权了两台，怀疑昨晚删除的是主力笔记本的授权，请问现在这种情况怎么解决？

**Reply by kennylo47** [reply]
fine 重装了一次 终于又变成pro版了 希望这次真的结束这个麻烦
